src/pages/ws/ui/cell-gene.tsx

```diff
@@ -16,7 +17,8 @@ export const CellGene = observer(
     const value = get(cell, 'value[0]', []) as string[]
     const rowIndex = get(cell, 'row.index', -1)
 
-    const iconColor = datasetStore?.wsRecords?.[rowIndex]?.cl.split('-')[0]
+    const records = toJS(datasetStore?.wsRecords)
```

Comment:
why do you need toJS here?

Reply:
otherwise we have a plenty out-of-bounce warning in console